.gitignore

```diff
@@ -101,7 +101,6 @@ __pypackages__/
 .env
 .env.*
 !.env.template
-!.env.dev
```

Comment:
why this change? why delete .env.dev? if deleted, every person starting on the project will be tripped up by its non-existence?

Reply:
The problem is that it now needs to contain our ORCID secrets. I plan to share my .env.dev file, but I don't see another way around it. What do you think?

Reply:
what if we have a .env.dev with a blank ORCID secret? what happens after this change in a dev instance with such an .env.dev? if most of the functionality still works, I think it's preferable to maintain that